mod_reforged/hooks/items/item_container.nut

```diff
@@ -0,0 +1,8 @@
+::Reforged.HooksMod.hook("scripts/items/item_container", function(q) {
+// New Functions
+	q.getWeight <- function( _slots = null )
```

Comment:
This can be reduced to `return -this.getStaminaModifier(_slots)` because that function should _always_ return a negative number.

For now I think addition of this function is not good, we should wait until we have a proper weight system. This is only really used in one place right now and that place can instead use `-getStaminaModifier(_slots)` instead.